adapters/stroeerCore/stroeercore.go

```diff
@@ -0,0 +1,114 @@
+package stroeerCore
```

Comment:
Please change the package name in all files to `stroeercore` (all lowercase) to follow Go conventions.

Reply:
Ah, right. Darn. We are case sensitive in most of the code for bidder names. I was hoping to keep it all lowercase going forward, but since you already have a mixed case name for PBJS it's a bid cumbersome. Ok. I may change this package name constraint in the future, would you be cool if we change the package name later on to be all lower case?

Reply:
Yep, no problem :)